CHANGELOG.md

```diff
@@ -77,6 +77,14 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * New `git.sign-on-push` config option to automatically sign commits which are being
   pushed to a Git remote.
 
+* Publish `config-schema.json` in the documentation site. Users can now reference
+  the schema in their `~/.config/jj/config.toml` file:
+
+```toml
+# Get editor completions based on the config schema
+"$schema" = "https://jj-vcs.github.io/jj/latest/config-schema.json"
```

Comment:
Ideally, users would be able to target the version of the tool they have, but that's probably not always possible (i.e. git compiled versions?) We could figure that out later on, in any case!

Reply:
Once we have it deployed for a tagged release we can start updating to use the version (the same way the rest of the docs are versioned), it just won't work for any versions prior (unless we have a system for back porting that kind of thing?).